packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -50,6 +63,76 @@ const TextStyle _kLargeTitleTextStyle = TextStyle(
   color: CupertinoColors.black,
 );
 
+// There's a single tag for all instances of navigation bars because they can
+// all transition between each other (per Navigator) via Hero transitions.
+const _HeroTag _heroTag = _HeroTag();
+
+class _HeroTag {
+  const _HeroTag();
+  // This class exists because the Gallery smoke test wants all widgets from
+  // debugDumpApp() to have a toString();
```

Comment:
It's not because the test exists that the class exists, it's because we want the user to be able to determine what the tag is.